benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -824,16 +912,12 @@ msgstr ""
 "inscribirse si elige cambiar la tarjeta que usa para pagar el servicio de "
 "tránsito."
 
-msgid ""
-"You were not charged anything today. Thank you for using Cal-ITP Benefits!"
-msgstr "No se le cobró nada hoy. ¡Gracias por usar Cal-ITP Benefits!"
-
 msgid ""
 "You were not charged anything today. When boarding an MST bus, tap this card "
 "and you will be charged a reduced fare. You will need to re-enroll if you "
 "choose to change the card you use to pay for transit service."
 msgstr ""
-"No se le cobró nada hoy. Al abordar un autobús de CST, toque el lector con "
+"No se le cobró nada hoy. Al abordar un autobús de MST, toque el lector con "
```

Comment:
Small bug in our existing translation -- we had the `CST` slug in the `MST` copy!